RandomXSharp/Internals/LibRandomx.cs

```diff
@@ -0,0 +1,23 @@
+using AdvancedDLSupport;
+
+namespace RandomXSharp.Internals
+{
+    internal static class LibRandomx
+    {
+        private static ILibRandomx? _instance = null;
+
+        public static ILibRandomx Instance
+        {
+            get
+            {
```

Comment:
IMO, `LibRandomx.Instance` can be a matter when concurrent accessing to it. how about using `static` block for this initalization?

Reply:
Or, we can use [`Lazy<T>`](https://docs.microsoft.com/en-us/dotnet/api/system.lazy-1?view=netstandard-2.0).

Reply:
Made its initialization atomic using the static constructor: <10bc794c123822e67b521aa8f536831f521129e3>.